src/injectable/electron-mocks.js

```diff
@@ -57,23 +59,40 @@ module.exports = function install (config, testPageUrl) {
 
     ipc.connect();
 
-    var { BrowserWindow, Menu, dialog } = require('electron');
+    var { Menu, dialog, app } = require('electron');
 
     var { WebContents } = process.atomBinding('web_contents');
 
-    var origLoadURL = BrowserWindow.prototype.loadURL;
-
+    var origLoadURL = WebContents.prototype.loadURL;
 
     function stripQuery (url) {
         return url.replace(URL_QUERY_RE, '');
     }
 
-    BrowserWindow.prototype.loadURL = function (url) {
+    function isFileProtocol (url) {
+        return url.indexOf('file:') === 0;
+    }
+
+    function getCustomAppPathDir () {
+        let customAppPath = null;
```

Comment:
`config.appPath` represents the JS file used as an entry point here. It's better to use the original `appPath` passed by the user.